CHANGELOG.md

```diff
@@ -2,10 +2,11 @@
 
 ### Version 0.11.0
 
+* `WikiPage::getSection()` now includes subsections by default; disabling the new $includeSubsections option reverts to the old behavior of returning only the text until the first subsection ([#55])
 * Supports a section name (in addition to an index) in `WikiPage::setText()` and `WikiPage::setSection()` ([#45])
-* Improved section processing in `WikiPage::getText()` ([#33], [#37], [#50])
+* Improves section processing in `WikiPage::getText()` ([#33], [#37], [#50])
```

Comment:
I agree on the consistency. My preference would be to the imperative (which the old versions of these lines could be read as, if interpreted in the passive voice) over either past or present tense (I.e. a non-personal description of the change itself, rather than of what the change author intended). The first opposing opinion you linked to argues against present tense, but makes no mention of imperative, and the second one also makes no arguments against imperative. What do you think?

As for following the Keep A Changelog format, I'm all for it 😄, but let's do that in a separate PR.